webview-ui/src/components/ui/standard-tooltip.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from "react"
+import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from "./tooltip"
+
+interface StandardTooltipProps {
+	children: React.ReactNode
+	content: React.ReactNode
+	side?: "top" | "right" | "bottom" | "left"
+	align?: "start" | "center" | "end"
+	sideOffset?: number
+	className?: string
+	asChild?: boolean
+}
+
+/**
+ * StandardTooltip component that enforces consistent 300ms delay across the application.
+ * This component wraps the Radix UI tooltip with a standardized delay duration.
+ */
+export function StandardTooltip({
+	children,
+	content,
+	side = "top",
+	align = "center",
+	sideOffset = 4,
+	className,
+	asChild = true,
+}: StandardTooltipProps) {
+	return (
+		<TooltipProvider delayDuration={300}>
```

Comment:
The hardcoded 300ms delay is a magic number. Could we extract it into a named constant to make it easier to manage and document?

```suggestion
const STANDARD_TOOLTIP_DELAY = 300;

export function StandardTooltip({
	// ... props
}: StandardTooltipProps) {
	return (
		<TooltipProvider delayDuration={STANDARD_TOOLTIP_DELAY}>
```

This would make it clearer what the value represents and easier to update if needed.